devices/leviton.js

```diff
@@ -91,10 +91,12 @@ module.exports = [
         model: 'ZS057-D0Z',
         vendor: 'Leviton',
         description: 'Wall switch, 0-10V dimmer, 120-277V, Lumina™ RF',
-        extend: extend.light_onoff_brightness({disableEffect: true, noConfigure: true}),
-        fromZigbee: [fz.brightness, fz.identify, fz.lighting_ballast_configuration, fz.level_config],
-        toZigbee: [tz.light_onoff_brightness, tz.ballast_config, tz.level_config],
-        exposes: [e.light_brightness(),
+        // Do not use extend.light_onoff_brightness because that has fz.ignore_basic_report,
+        // which is what tells us when the physical switch is used
+        fromZigbee: [fz.on_off, fz.on_off_via_brightness, fz.brightness, fz.level_config, fz.identify,
```

Comment:
Above this line, can you try adding `meta: {disableDefaultResponse: true},` and see if that solves the connection issues?

Reply:
This helped, however it exposed another problem that I _think_ was caused by `tz.level_config` so I removed it. It's working kind of OK now: https://github.com/Koenkk/zigbee-herdsman-converters/pull/4834/commits/c5cf4b0f0f00f42d10aebf1d13062932db355aee

Very open to suggestions on the reporting thing that feels like a hack...